### PR TITLE
fix(combo-button): placement in scrollable parent

### PR DESCRIPTION
**Pull request by j1mie**
## Affected issues

- Resolves partially https://github.com/carbon-design-system/ibm-security/issues/178

## Proposed changes

- Tweaks placement, creating a nested group to create the layout that the menu item's width is derived from

## Testing instructions

- Add a `ComboButton` inside a div without a fixed width
- Menu should no longer occupy 100% of wrapper div.

### CODE REVIEW

**Comment by SimonFinney** [comment]
@j1mie As per discussion this morning, should we hold out on merging this until you investigate further?

**Comment by j1mie** [comment]
Hey @SimonFinney - this PR resolves the issue of the `ComboButton` in a scrollable parent. I was having a separate issue with creating a floating menu in a table toolbar, which I would consider separate to the bug referenced.

File: src/components/ComboButton/ComboButton.js [hunk 1]
```diff
@@ -124,34 +124,36 @@ const ComboButton = ({ children, className, direction }) => {
...
-          onClose={() => setIsOpen(false)}
-          onOpen={() => setIsOpen(true)}
-          renderIcon={() =>
-            isOpen ? overflowIcon(ChevronUp16) : overflowIcon(ChevronDown16)
-          }
-        >
-          {overflowMenuItemWithProps}
-        </OverflowMenu>
-      )}
+      <div className={`this ${namespace}__group`}>
```
Comment (SimonFinney) [inline src/components/ComboButton/ComboButton.js]: - [ ] Would you mind removing the `this` class if it's not doing anything?

Comment (jendowns) [reply src/components/ComboButton/ComboButton.js]: @SimonFinney I disagree on this... I feel like before, we've talked about attaching classes to wrappers, even if *we* arent' necessarily styling said wrappers, in order to allow end users to more easily target those wrappers, if needed. I believe this is Carbon's approach as well.

Comment (j1mie) [reply src/components/ComboButton/ComboButton.js]: Oops. Typo. Sorry!

**Comment by jendowns** [comment]
@j1mie could you run `yarn test -u` one more time? 😅

**Comment by SimonFinney** [comment]
Thanks @j1mie !

**Pull request merged**